sphinx/builders/latex/constants.py

```diff
@@ -161,7 +171,8 @@
         'latex_engine': 'platex',
         'babel':        '',
         'classoptions': ',dvipdfmx',
-        'fontpkg':      '\\usepackage{times}',
+        'fontpkg':      PDFLATEX_DEFAULT_FONTPKG,
+        'fontsubstitution': '',
```

Comment:
I am hoping that this is ok for platex: it does not need font substitution to allow individual Greek letters (if I recall correctly from testing 2 years ago)